inductiva/_cli/tasks.py

```diff
@@ -1,22 +1,61 @@
 """Tasks CLI subcommands."""
 from inductiva import tasks, utils
 from inductiva import _cli
+from inductiva.utils import format_utils
+from typing import Sequence
+
+
+def _data_of_tasks(list_of_tasks: Sequence["inductiva.tasks.Task"]) -> str:
```

Comment:
this method is tightly bound to the `list_tasks` method. The content can be moved to that method.

Reply:
Done